pkg/state/factory.go

```diff
@@ -121,12 +121,18 @@ func newNicClusterPolicyStates(k8sAPIClient client.Client, scheme *runtime.Schem
 	nvIpamCniState, err := NewStateNVIPAMCNI(
 		k8sAPIClient, scheme, filepath.Join(manifestBaseDir, "state-nv-ipam-cni"))
 	if err != nil {
-		return nil, errors.Wrapf(err, "failed to create ib-kubernetes State")
```

Comment:
thanks for the fix!

Reply:
np, while not related to this PR, it just bothered me :) plus im the one who originally added it.